consensus/consensus_v2.go

```diff
@@ -473,12 +470,19 @@ func (consensus *Consensus) Start(
 
 			case viewID := <-consensus.commitFinishChan:
 				consensus.getLogger().Debug().Msg("[ConsensusMainLoop] commitFinishChan")
+
 				// Only Leader execute this condition
 				func() {
+					if time.Now().Before(blockTime) {
```

Comment:
```
if n := time.Now(); n.Before(blockTime) {

time.Sleep(blockTime.Sub(n)) 

}
```

unless you are intentionallyy making a fresh `time.Now()`?

Reply:
good point. done.